Create getlivery.vbs

HomeFries94 opened the pull request:
Proper command line usage of this script is:
    getlivery [-beta] [filename]

where the -beta argument will parse the DCS World Openbeta main folder and user folder
and filename is the name of the output file.
If no path is specified, the file will be output to the DCS user folder (main or openbeta depending on -beta argument).
If no filename is specified, the output will be "DCS_Liveries.txt" in the DCS user folder.
If you want the filename created in the same folder as this script, use the following line:
    getlivery [-beta] .\filename

IMPORTANT: if you use the -beta argument, it must come before the filename.

jboecker commented:
You committed the script to the top level directory. Can you change your pull request to put it in the scripts/ subfolder?

(Using `git gui`, select the "amend last commit", un-stage getlivery.vbs by clicking its entry in the "Staged Changes" list, move the file to scripts/getlivery.vbs, re-stage, then click Commit. When pushing to GitHub, you'll have to check the "force" checkbox because you just re-wrote history, which one should only do as long as nobody else has pulled from that branch.)

While you are at it, you can also add "closes #8" somewhere in your commit message: that will tell GitHub to close issue #8 as soon as that commit is merged into the master branch.

HomeFries94 committed:
Create getlivery.vbs
add scripts/getlivery.vbs to a generate livery list ready for upload to an instance
closes #8

Proper command line usage of this script is:
	getlivery [-beta] [filename]

where the -beta argument will parse the DCS World Openbeta main folder and user folder
and filename is the name of the output file.
If no path is specified, the file will be output to the DCS user folder (main or openbeta depending on -beta argument).
If no filename is specified, the output will be "DCS_Liveries.txt" in the DCS user folder.
If you want the filename created in the same folder as this script, use the following line:
	getlivery [-beta] .\filename

IMPORTANT: if you use the -beta argument, it must come before the filename.

HomeFries94 commented:
Done.

jboecker commented:
Awesome, thanks!
